### PR TITLE
Add Feature that makes the cap still give reward but exponentially less

### PR DESCRIPTION
**Pull request by Narimm**

### CODE REVIEW

**Comment by Narimm** [comment]
The equation produced a exponential degrading curve that reduced income exponentially the factor controls the rate

File: src/main/java/com/gamingmesh/jobs/container/JobsPlayer.java [hunk 1]
```diff
@@ -1,961 +1,974 @@
...
+	if (data.IsReachedLimit(type, value == null ? 0 : value)) {
+	    if (player.isOnline() && !data.isInformed() && !data.isReseted()) {
+			if(Jobs.getGCManager().useMaxPaymentCurve) {
+				player.sendMessage(Jobs.getLanguage().getMessage("command.limit.output.reached" + type.getName().toLowerCase() + "limit"));
+				player.sendMessage(Jobs.getLanguage().getMessage("command.limit.output.reached" + type.getName().toLowerCase() + "limit2"));
+				player.sendMessage(Jobs.getLanguage().getMessage("command.limit.output.reached" + type.getName().toLowerCase() + "limit3"));
+			}else {
+				player.sendMessage(Jobs.getLanguage().getMessage("command.limit.output.reached" + type.getName().toLowerCase() + "limit"));
+				player.sendMessage(Jobs.getLanguage().getMessage("command.limit.output.reached" + type.getName().toLowerCase() + "limit2"));
+				data.setInformed();
```
Comment (montlikadani) [inline src/main/java/com/gamingmesh/jobs/container/JobsPlayer.java]: Hmm.. Double?

File: src/main/java/com/gamingmesh/jobs/container/JobsPlayer.java [hunk 2]
```diff
@@ -1,961 +1,974 @@
...
+			}
+			data.setInformed();
```
Comment (montlikadani) [inline src/main/java/com/gamingmesh/jobs/container/JobsPlayer.java]: same

**Comment by Narimm** [comment]
Removed the double up!

**Pull request merged**